lib/platform/common.ts

```diff
@@ -96,12 +96,7 @@ export interface Issue {
   title?: string;
 }
 
-export type BranchStatus =
-  | 'pending'
-  | 'success'
-  | 'failed'
-  | 'running'
-  | 'failure';
+export type BranchStatus = 'green' | 'yellow' | 'red';
```

Comment:
@viceice there's definitely no way to use our constants here or otherwise correlate them, right?

Reply:
yes, but you could move this to constants and declare them as `export const BRANCH_STATUS_GREEN : BranchStatus = 'green';`.

This is the only why i'm currently aware of.